apps/trackers/jira/query.py

```diff
@@ -129,7 +129,10 @@ def generate_labels(self):
         )
 
         # If all affects are NEW and if this tracker is new, add label validation-requested.
-        if not self.tracker.external_system_id:
+        if (
+            not self.tracker.external_system_id
+            or "validation-requested" in self.tracker.meta_attr.get("labels", [])
+        ):
```

Comment:
OK, I'll make it stateless - add the label whenever all affects are NEW, drop the label whenever they aren't.

Relevant discussion: https://github.com/RedHatProductSecurity/osidb/pull/420#discussion_r1463393381

Reply:
Please review commit [6012a47](https://github.com/RedHatProductSecurity/osidb/pull/420/commits/6012a478a616279dbb3998a878c116c898e64b29)

Reply:
I'm still leaving the storage of labels in meta_attr because it makes the test more descriptive and close to reality and the tracking would have to be added in OSIDB-1440 anyway (i.e. removing it in PR420 would add extra work).